### PR TITLE
feat: bump OZ version (0.4.9)

### PR DESCRIPTION
**Pull request by 0xClandestine**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,15 +1,9 @@
...
 [submodule "lib/ds-test"]
 	path = lib/ds-test
 	url = https://github.com/dapphub/ds-test
 [submodule "lib/forge-std"]
 	path = lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
-[submodule "lib/openzeppelin-contracts"]
-	path = lib/openzeppelin-contracts
-	url = https://github.com/OpenZeppelin/openzeppelin-contracts
 [submodule "lib/openzeppelin-contracts-v4.9.0"]
```
Comment (gpsanant) [inline .gitmodules:7]: can we get rid of the `-v4.9.0` everywhere

Comment (gpsanant) [reply .gitmodules:7]: oh i see you did this in the remapping, would be nice to change the folder name too

**Comment by 0xClandestine** [comment]
bindings likely need fixed

File: .gitmodules (hunk 1 above)
Comment (0xClandestine) [reply .gitmodules:7]: Wondering how to properly rename a submodule?

File: bin/storage-report.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+mkdir storage-report
+for file in $(find src -name '*.sol'); do
+    contract_name=$(basename "$file" .sol)
```
Comment (gpsanant) [inline bin/storage-report.sh:3]: could you do it only for the `src/contracts/*.sol`?

**Pull request closed without merging**